scormxblock/scormxblock.py

```diff
@@ -156,7 +156,7 @@ def updoad_all_content(temp_directory, fs):
         s3_upload(all_content, temp_directory, dest_dir)
     else:
         # The raw number of files is going to make this request time out. Use celery instead
-        s3_upload.apply_async((all_content, temp_directory, dest_dir), serializer='pickle')
+        s3_upload.delay(all_content, temp_directory, dest_dir)
```

Comment:
is it necessary to change apply_async for delay? @luismorenolopera

Reply:
no.  but now as no extra arguments are used it seems correct to use delay

Reply:
Ok.